twisted/conch/ssh/keys.py

```diff
@@ -1121,6 +1191,11 @@ def _toString_AGENTV3(self):
             elif self.type() == 'DSA':
                 values = (data['p'], data['q'], data['g'], data['y'],
                           data['x'])
+            # I assume this is what's needed
+            # But commented out because it hasn't been tested yet.
+            elif self.type() == 'EC':
+                raise UnsupportedAlgorithm("toString() does not support Elliptic Curves yet.")
```

Comment:
My point is that the PR needs to include the link, which means the ticket needs to be filed first :).  I realize this is a little awkward, but just include some context in the ticket's description, and if the PR doesn't land or the ticket becomes invalid, someone else can close it later.